File: API/service/classes/election_year_service.py
from repository.classes.election_year_repository import ElectionYearRepository


class ElectionYearService:
    def __init__(self):
        self.repository = ElectionYearRepository()

    def get_election_year_details(self, election_year_id):
        return self.repository.get_election_year_details(election_year_id)

File: UI/web-app/src/app/metadata-4p/electionyear/electionyear.service.ts
import { Injectable } from '@angular/core';
import { Http, Response, Headers, RequestOptions } from '@angular/http';
import {Observable} from 'rxjs/Rx';


@Injectable()
export class ElectionYearService {
    private electionyeardetailsurl = 'http://localhost:8000/ElectionYear';
    constructor(private http: Http) {
    }
    getElectionYearDetails(electionYearId: string): Observable<any> {
        return this.http.get(this.electionyeardetailsurl + '/?electionYearId=' + electionYearId);
    }
}
